docs/development-guide/DevelopmentGuide.md

```diff
@@ -125,9 +125,9 @@ See [here](PluginPipelinePerformanceResults.md#failover-performance-with-differe
 ![](../images/jdbc_wrapper_postgresql_efm_6000_1000_1.png)
 See [here](PluginPipelinePerformanceResults.md#enhanced-failure-monitoring-performance-with-different-failure-detection-configuration) for a more detailed performance breakdown.
 
-#### Read-Write Splitting and Reader Load Balancing Performance Tests
+#### Read-Write Splitting Performance Tests
 
-The diagram in this section shows the AWS JDBC Driver's performance when using the read-write splitting plugin, with or without reader load balancing. This test sets up a large number of connections in parallel; the initial connection is to the writer but will be changed to various reader instances if reader load balancing is enabled. The test executes a long query many times to simulate heavy queries.
+The diagram in this section shows the AWS JDBC Driver's performance when using the read-write splitting plugin. This test sets up a large number of connections in parallel; the initial connection is to the writer. The test executes a long query many times to simulate heavy queries.
 
 The average overhead time is measured as the average time difference between running the entire test with the read-write plugin and running the entire test without any plugins. The baseline overhead time is 0 because there are no plugins in this scenario and thus there is no plugin overhead.
 
```

Comment:
We will have to change the connected image as well since it has a bar for r/w with load balancing. Remind me where the test code for these results? I'm not sure if we should run the test with internal connection pooling instead of load balancing